sh/prepareData_R2HDM.py

```diff
@@ -15,28 +15,45 @@ def convert(InputFile, OutputFile):
     print(f"Reading {InputFile}.")
     print(f"Output is saved to {OutputFile}.")
 
-    HasIndexCol = False
-    Separator = "\t"
-    Type = "yuktype"
-    Lambda1 = "L1"
-    Lambda2 = "L2"
-    Lambda3 = "L3"
-    Lambda4 = "L4"
-    Lambda5 = "L5"
-    tanbeta = "tbeta"
-    m12squared = "m12sq"
+Seperator = "\t"
+InputFILE = "../example/R2HDM_Input.dat"
+InputFILE = "wI.csv"
+OutputFILE = "R2HDM_Ordered.dat"
+Type = "yuktype"
+Lambda1 = "L1"
+Lambda2 = "L2"
+Lambda3 = "L3"
+Lambda4 = "L4"
+Lambda5 = "L5"
+tanbeta = "tbeta"
+m12squared = "m12sq"
 
     with open(InputFile, "r") as file:
         df = pd.read_csv(file, index_col=HasIndexCol, sep=Separator)
```

Comment:
Why is this indented?

Reply:
I propose to fix the `sh/prepareData_*.py` files by merging #165. Ok?